ovs-p4rt/sidecar/common/ovsp4rt_utils.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OVSP4RT_UTILS_H_
+#define OVSP4RT_UTILS_H_
+
+#include <netinet/in.h>
+
+#include <cstdint>
+#include <string>
+
+#include "p4/config/v1/p4info.pb.h"
+
+namespace ovs_p4rt {
+
+extern std::string CanonicalizeIp(const uint32_t ipv4addr);
```

Comment:
Declarations being introduced here. Did they not exist somewhere before or is there a cleanup needed someplace?

Reply:
1. No function prototypes were required previously because the utility functions were in the same file as the functions that called them.
2. This PR moves the functions to a different source file, so that is no longer true.
3. We need a header file that defines prototypes for the utility functions, so the code in ovsp4rt.cc can continue to call them.
4. This PR defines a header file that defines the interface to the new file.